src/components/Footer.tsx

```diff
@@ -42,7 +42,7 @@ const Footer = () => (
               isExternal
               href="https://vercel.com/?utm_source=octoclairvoyant-team&utm_campaign=oss"
             >
-              <Image src={poweredByVercelLogo} />
+              <Image alt="Powered by Vercel" src={poweredByVercelLogo} />
```

Comment:
```suggestion
              <Image alt="Powered by Vercel logo" src={poweredByVercelLogo} />
```

I think having the `logo` part in the alt text is clearer... 😄